addOns/formhandler/src/main/java/org/zaproxy/zap/extension/formhandler/FormHandlerParam.java

```diff
@@ -56,16 +56,34 @@ public class FormHandlerParam extends VersionedAbstractParam {
     private static final String CONFIRM_REMOVE_TOKEN_KEY =
             FORM_HANDLER_BASE_KEY + ".confirmRemoveField";
 
-    private static final Map<String, String> DEFAULT_KEY_VALUE_PAIRS = new HashMap<>();
-
-    static {
-        DEFAULT_KEY_VALUE_PAIRS.put("color", "#ffffff");
-        DEFAULT_KEY_VALUE_PAIRS.put("email", "foo-bar@example.com");
-        DEFAULT_KEY_VALUE_PAIRS.put("name", "ZAP");
-        DEFAULT_KEY_VALUE_PAIRS.put("password", "ZAP");
-        DEFAULT_KEY_VALUE_PAIRS.put("phone", "9999999999");
-        DEFAULT_KEY_VALUE_PAIRS.put("url", "https://www.example.com");
-    }
+    private static final List<FormHandlerParamField> DEFAULT_FIELDS =
+            List.of(
+                    new FormHandlerParamField("color", "#ffffff"),
+                    new FormHandlerParamField("email", "foo-bar@example.com"),
+                    new FormHandlerParamField("name", "ZAP"),
+                    new FormHandlerParamField("password", "ZAP"),
+                    new FormHandlerParamField("phone", "9999999999"),
+                    new FormHandlerParamField("url", "https://www.example.com"),
+                    new FormHandlerParamField(
+                            "(?i)_?back[-_]?(?:link|uri|url)?",
+                            "https://www.example.com",
+                            true,
+                            true),
+                    new FormHandlerParamField("(?i)_?bg[-_]?colou?r", "#FFFFFF", true, true),
+                    new FormHandlerParamField("(?i)_?query|find|keyword", "ZAP", true, true),
```

Comment:
Add name and password too?

Reply:
I thought about doing that but then thought:
1. ZAP isn't a  very password'ish string.
2. We might be more likely to lock people out.

Reply:
Maybe in the next PR when I add datafaker I'll do a password one, with a string that actually seems password'ish. For username maybe we go zapzap? 

Should I default them to off?